pages/1.9/security/ent/perms-reference/index.md

```diff
@@ -42,6 +42,7 @@ Here are the available CRUD actions (`create`, `read`, `update`, and `delete`).
 | `dcos:adminrouter:ops:system-logs` <br>Controls access to [System logs API](/1.9/api/master-routes/#system).                                                                                                                                                                      | x    |   |   |   |   |
 | `dcos:adminrouter:ops:system-metrics`<br> Controls access to [System metrics API](/1.9/api/master-routes/#system).                                                                                                                                                                | x    |   |   |   |   |
 | `dcos:adminrouter:package` <br> Controls access to the [Cosmos API](/1.9/api/master-routes/#cosmos), which provides access to the DC/OS Universe.                                                                                                                                 | x    |   |   |   |   |
+| `dcos:adminrouter:secrets` <br>  Controls access to Secrets UI.                                                                                                                                                                      | x    |   |   |   |   |
```

Comment:
I'm working on the assumption that there is no Secrets UI page in 1.9.

Reply:
Correct!